client/webserver/site/src/js/doc.ts

```diff
@@ -291,6 +296,15 @@ export default class Doc {
     return fullPrecisionFormatter(prec).format(value)
   }
 
+  static decodeRateStep (rateStepEnc: number, baseUnitInfo: UnitInfo, quoteUnitInfo: UnitInfo) {
+    const [qFactor, bFactor] = [quoteUnitInfo.conventional.conversionFactor, baseUnitInfo.conventional.conversionFactor]
+    return rateStepEnc / (RateEncodingFactor * qFactor / bFactor)
+  }
+
+  static rateStepDigits (rateStepDec: number) {
+    return Math.round(Math.log10(1 / rateStepDec))
+  }
```

Comment:
You might be onto something here using the conventional ratestep to determine the number of decimals to show. But, the argument is not just decoded (`rateStepDec`), it's conventional, so the argument name should reflect that. Also, this is only used once. Do we really need a method for this?